en/flight_modes/land.md

```diff
@@ -26,30 +26,132 @@ Landing is affected by the following parameters:
-<span id="COM_RC_STICK_OV"></span>[COM_RC_STICK_OV](../advanced_config/parameter_reference.md#COM_RC_STICK_OV) | The amount of stick movement that causes a transition to [Position mode](../flight_modes/position_mc.md) (if [COM_RC_OVERRIDE](#COM_RC_OVERRIDE) is enabled).
+<a id="MPC_LAND_SPEED"></a>[MPC_LAND_SPEED](../advanced_config/parameter_reference.md#MPC_LAND_SPEED) | The rate of descent during landing. This should be kept fairly low as the ground conditions are not known.
+<a id="COM_DISARM_LAND"></a>[COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND) | Time-out for auto disarm after landing, in seconds. If set to -1 the vehicle will not disarm on landing.
+<a id="COM_RC_OVERRIDE"></a>[COM_RC_OVERRIDE](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) | Controls whether stick movement on a multicopter (or VTOL in MC mode) causes a mode change to [Position mode](../flight_modes/position_mc.md). This can be separately enabled for auto modes and for offboard mode, and is enabled in auto modes by default.
+<a id="COM_RC_STICK_OV"></a>[COM_RC_STICK_OV](../advanced_config/parameter_reference.md#COM_RC_STICK_OV) | The amount of stick movement that causes a transition to [Position mode](../flight_modes/position_mc.md) (if [COM_RC_OVERRIDE](#COM_RC_OVERRIDE) is enabled).
 
 
-## Fixed Wing (FW)
+## Fixed-wing (FW)
 
-The vehicle will turn and land at the location at which the mode was engaged.
-RC stick movement is ignored.
+Fixed-wing landing via mav command is currently not supported.
+The landing mode is only accessible in missions.
+Please see QGroundControl docs for more information on [planning a fixed-wing landing pattern mission item](https://docs.qgroundcontrol.com/master/en/PlanView/pattern_fixed_wing_landing.html).
 
-Fixed wing landing logic and parameters are explained in the topic: [Landing (Fixed Wing)](../flying/fixed_wing_landing.md).
+### Landing Sequence
+
+The fixed-wing landing pattern has three components: a **landing descent orbit**, a **landing waypoint**, and a **landing approach slope**.
+
+On engaging a landing pattern, the following sequence is initiated:
+
+1. **Fly to landing location**: The aircraft flies at the current altitude to the landing descent orbit.
+2. **Descend to approach altitude**: On reaching the orbit, the vehicle will descend to the approach altitude.
+   Once reaching the approach altitude, the airframe will continue on the orbit until reaching the final approach exit (tangential exit towards landing waypoint).
+4. **Final approach**: The aircraft will follow the landing approach slope until the flare altitude is reached.

```

Comment:
It is much better if this title "Final approach" matches the heading it maps too, and even links. So I think this should be "Landing approach" (I.e. I am guessing this is what you talk about in the section below).